src/extension/tab/hook.ts

```diff
@@ -256,8 +262,12 @@ function initializeHook() {
     sendHookDataToDevTools("connectToDevtools");
   }
 
+  function handleClientMessage(message: ClientDevtoolsMessage) {
+    tab.send(message);
```

Comment:
I'm very quickly realizing this "fire and forget" is too simplistic for something like Reactive vars. If the user never clicks into devtools, these messages get easily lost, which means the devtools scripts won't be able to pick up registered reactive vars.

Instead, we should do similar to queries/mutations and keep track of registered vars in a variable somewhere in this script. The devtools could then ask for any registered vars on initialization and sync that over.